jwt/decoder_options_test.go

```diff
@@ -0,0 +1,36 @@
+package jwt
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDecoderOptions(t *testing.T) {
+	b, err := os.ReadFile(filepath.Clean(testAuthJwks))
+	assert.Nil(t, err)
+	validJwks := string(b)
+
+	i := 0
+	jwks := func() string {
```

Comment:
```suggestion
// This will be called each time the cache is refreshed and they we can assert i has been incremented the correct number of times below.
	jwks := func() string {
```
I struggled at first to understand what this test and encoder were doing so thought I'd add a comment?